lib/saml_config.rb

```diff
@@ -2,12 +2,10 @@
 
 module Saml
   class Config
-    def initialize
-      @@settings ||= build_settings.freeze
-    end
+    attr_reader :settings
 
-    def settings
-      @@settings
+    def initialize
+      @settings ||= build_settings.freeze
```

Comment:
ah gotcha. Rubocop was complaining about usage of a class var at all. Perhaps I should disable that cop?